cPP_DSC.adoc

```diff
@@ -4019,203 +4015,391 @@ The dependencies between SFRs implemented by the TOE are addressed as follows.
 |Rationale Statement
 
 |FCS_CKM.1 
-|[FCS_CKM.2, FCS_CKM.5, FCS_CKM_EXT.7 or FCS_COP.1] 
+|[_FCS_CKM.1/AKG_, _FCS_CKM.1/SKG_, FCS_CKM.2, FCS_CKM.5, FCS_CKM_EXT.7, _FCS_CKM_EXT.8_ or FCS_COP.1] 
+
+FCS_CKM.3
 
 [FCS_RBG.1 or FCS_RNG.1]
 
-FCS_CKM.6 
-|FCS_CKM.2, FCS_CKM.6, FCS_CKM_EXT.7, FCS_COP.1 and FCS_RBG.1 are required by the PP.
+FCS_CKM.6
+|FCS_CKM.1/AKG - (selection-based) included
+
+FCS_CKM.1/SKG - (selection-based) included
+
+FCS_CKM.2 - included
+
+FCS_CKM.3 is satisfied by FAU_STG_EXT.1 which controls access to keys - included
+
+FCS_CKM_EXT.7 - included
+
+FCS_CKM_EXT.8 - (selection-based) included
+
+FCS_COP.1 - included
+
+FCS_RBG.1 - included
+
+FCS_RNG.1 is satisfied by FCS_RBG.1 - included
+
+FCS_CKM.6 - included
 
 |FCS_CKM.2
-|[FDP_ITC.1, FDP_ITC.2, FCS_CKM.1 or FCS_CKM.5] 
+|[FDP_ITC.1, FDP_ITC.2, FCS_CKM.1, FCS_CKM.5 or _FCS_COP.1_] 
+|FDP_ITC.1 is satisfied by FDP_ITC_EXT.1 related to a mechanism by which key data can be imported into the TSF - included
+
+FDP_ITC.2 is satisfied by FDP_ITC_EXT.2 related to a mechanism by which key data can be imported into the TSF - included
+
+FCS_CKM.1 - included
+
+FCS_CKM.5 - (selection-based) included
+
+FCS_COP.1/KeyEncap - (selection-based) included
+
+FCS_COP.1/KeyWrap - (selection-based) included
+
+FCS_COP.1/AEAD - (selection-based) included
+
+|FCS_CKM.6 
+|[FDP_ITC.1, FDP_ITC.2, FCS_CKM.1, FCS_CKM_EXT.3, FCS_CKM.5, FCS_CKM_EXT.7 or FCS_CKM_EXT.8] 
```

Comment:
From what I read in CC:2022, FCS_CKM_EXT.3, FCS_CKM.5, FCS_CKM_EXT.7 or FCS_CKM_EXT.8 are not dependencies of FCS_CKM.6. Should they be italicized?

Reply:
@slpotte I added these because they are in the catalog. There is a note (to Frank) to add text as to why they are added. I expect they are added because at least some of them relate to how the keys are protected in the first place, so they provide some sort of protection and justification as to why it is OK that keys are deleted in a specific manner.

If this isn't available, I don't have a problem, for now, just taking out those and leaving only the raw CC:2022 dependencies.

Reply:
For now I am going to remove these dependencies